<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CotizacionesForm</class>
 <widget class="QWidget" name="CotizacionesForm">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="formLayout">
     <item row="0" column="0"><widget class="QLabel" name="labelID"><property name="text"><string>ID</string></property></widget></item>
     <item row="0" column="1"><widget class="QLineEdit" name="inputID"/></item>
     <item row="1" column="0"><widget class="QLabel" name="labelCodigo"><property name="text"><string>Código</string></property></widget></item>
     <item row="1" column="1"><widget class="QLineEdit" name="inputCodigo"/></item>
     <item row="2" column="0"><widget class="QLabel" name="labelMonto"><property name="text"><string>Monto</string></property></widget></item>
     <item row="2" column="1"><widget class="QLineEdit" name="inputMonto"/></item>
     <item row="3" column="0"><widget class="QLabel" name="labelFechaEmision"><property name="text"><string>Fecha Emisión</string></property></widget></item>
     <item row="3" column="1"><widget class="QLineEdit" name="inputFechaEmision"/></item>
     <item row="4" column="0"><widget class="QLabel" name="labelRutCliente"><property name="text"><string>RUT Cliente</string></property></widget></item>
     <item row="4" column="1"><widget class="QLineEdit" name="inputRutCliente"/></item>
     <item row="5" column="0"><widget class="QLabel" name="labelBuscar"><property name="text"><string>Buscar</string></property></widget></item>
     <item row="5" column="1"><widget class="QLineEdit" name="inputBuscar"/></item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="botonesLayout">
     <item><widget class="QPushButton" name="btnAgregar"><property name="text"><string>Agregar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnEditar"><property name="text"><string>Editar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnEliminar"><property name="text"><string>Eliminar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnBuscar"><property name="text"><string>Buscar</string></property></widget></item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tablaCotizaciones"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
